test/utils/logger.test.js

```diff
@@ -122,6 +130,73 @@ describe('#getLogLevel', () => {
   });
 });
 
+const id = faker.random.uuid();
+
+const PIIData = {
+  id,
+  full_name: faker.name.findName(),
+  email: faker.internet.email(),
+  phone: faker.phone.phoneNumber(),
+  accept_language: 'en-CA',
+  address: {
+    address: faker.address.streetAddress(),
+    address_2: faker.address.streetAddress(),
+    city: faker.address.city(),
+    country_code: 'CA',
+    country_subregion_code: 'AB',
+    postal_code: 'S0S 6V0',
+  },
+  gift: false,
+  ip_address: '::ffff:127.0.0.1',
+};
+
+const filteredPIIData = {
+  id,
+  full_name: '[REDACTED]',
+  email: '[REDACTED]',
+  phone: '[REDACTED]',
+  accept_language: 'en-CA',
+  address: '[REDACTED]',
+  gift: false,
+  ip_address: '[REDACTED]',
+};
+
+describe('#filterLogData', () => {
```

Comment:
Should we move these into the `filterLogData` scope?
```suggestion
describe('#filterLogData', () => {
	  const id = faker.random.uuid();
	  
	  const PIIData = {
	    id,
	    full_name: faker.name.findName(),
	    email: faker.internet.email(),
	    phone: faker.phone.phoneNumber(),
	    accept_language: 'en-CA',
	    address: {
	      address: faker.address.streetAddress(),
	      address_2: faker.address.streetAddress(),
	      city: faker.address.city(),
	      country_code: 'CA',
	      country_subregion_code: 'AB',
	      postal_code: 'S0S 6V0',
	    },
	    gift: false,
	    ip_address: '::ffff:127.0.0.1',
	  };
	  
	  const filteredPIIData = {
	    id,
	    full_name: '[REDACTED]',
	    email: '[REDACTED]',
	    phone: '[REDACTED]',
	    accept_language: 'en-CA',
	    address: '[REDACTED]',
	    gift: false,
	    ip_address: '[REDACTED]',
	  };


```

Reply:
Yeah good suggestion, it was originally, but the describe blocks for #filterLogData and #processLogData both use these objects...

An alternative is to move them to the fixtures folder as test helpers?

Reply:
The issue there is that I don't think they are used in any other tests, so may be premature to move them to a shared space 🤔

Reply:
Ah, okay, I missed that. Maybe we can wrap the 2 in a `log data` context? My primary concern is leaking scope. If we encapsulate everything, that eliminates the risk… What do you think?

Reply:
Hey I moved all the fixtures to the top of the file. I am reusing them in the new tests I added. Can you take another look?